packages/terra-form-field/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,15 @@
 # Terra Form Field Upgrade Guide
 
+## Changes from version 3 to version 4
+
+### Props
+
+#### Updated
+* `label` prop's type changed from node to string.
+ 
+##### Steps to Upgrade
+To upgrade from 3.0 to 4.0 each existing Field must be reviewed and updated to ensure only a string is being passed to the `labelText` prop. Solutions passing node data types need to be reworked to provide only a string.
```

Comment:
Should be **`label`** since form-field does not have `labelText`.

Reply:
Changed 83b428d